Jenkinsfile_CNP

```diff
@@ -66,5 +52,4 @@ withPipeline("java", product, component) {
   enableSlackNotifications(notificationsChannel)
   enableDockerBuild()
   installCharts()
-  setVaultName('cmc')
```

Comment:
you should probably also stop outputting the vault name as part of this change:
https://github.com/hmcts/cmc-claim-store/blob/master/infrastructure/output.tf#L1-L7